src/pages/sunhoLee/Login/Login.js

```diff
@@ -1,9 +1,123 @@
+import { formatWithCursor } from 'prettier';
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
+import '../Login/Login.scss';
+// import '../../../styles/common.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idValue: '',
+      pwValue: '',
+      isButtonOn: false,
+    };
+  }
+
+  handleIdInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      idValue: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      pwValue: e.target.value,
+    });
+  };
```

Comment:
```suggestion
  handlePwInput = e => {
    const { name, value } = e.target;
    this.setState({
      pwValue: value,
    });
  };
```
- 구조분해할당을 하고 나서 사용하지를 않고 있네요
- name property도 사용하고 있지 않습니다.
- name property를 이용해서 handlePwInput, handleIdInput 합칠 수 있어보입니다 시도해보세요

Reply:
대괄호[ ]가 있는거는 Computed property names라는 문법이 있는걸 확인했습니다!!!!

handleInput = e => {
    const { name, value } = e.target;
    this.setState({
      [name]: value,
    });
  };
이런식으로 바꿨슴돠!!